### PR TITLE
ddPCR report

### PR DESCRIPTION
**Pull request by Bella2347**
Updated metod description for RT-qPCR and ddPCR P190 and in doing so separated the method description from KIT. 

Added Tissue to the historical data file and report. If tissue is missing the column is just left empty.

### CODE REVIEW

File: ddPCR/ddPCR+RT-qPCR_createReport.py [hunk 1]
```diff
@@ -193,12 +214,28 @@ def calculateMAF( sample_data, sample ):
...
-
-	tableStr = '\n'.join(tableStr)
+		sampleName = fileTable["SampleName"][i]
+		if "Tissue" in fileTable:
+			tissue = fileTable["Tissue"][i]
+		else:
+			tissue = ""
+		date = fileTable["Date"][i].strftime("%Y-%m-%d")
+		data_column = fileTable[name_data_column][i]
+		LoD = fileTable["LoD"][i]
```
Comment (ErikBergman) [inline ddPCR/ddPCR+RT-qPCR_createReport.py:227]: Error handling here? Or do we want an explicit crash if `name_data_column` should be undefined?

File: customPackageSMD/variables.py [hunk 2]
```diff
@@ -116,3 +116,8 @@
 		'PAT' : 	{'name' : 'ZebraPat', 'labelType' : 'square'},
 		}
 
+TISSUE_DICT = {
```
Comment (ErikBergman) [inline customPackageSMD/variables.py:119]: All other variables here are in lowercase

**Review by ErikBergman — CHANGES_REQUESTED** [review]
Looking good. Many lines involve getting stuff from dicts without any error handling, but maybe that's accounted for?

**Comment by Bella2347** [comment]
Changes have been tested for all ddPCR and RT-qPCR analysis. Documented on ReQtest: https://secure.reqtest.com/app/tests/test-run/view/view-test-run-details/757441

**Pull request merged**